source/Octopus.Configuration/Octopus.Configuration.csproj

```diff
@@ -10,6 +10,8 @@
     <Description>Configuration interfaces for Octopus, an opinionated deployment solution for .NET applications</Description>
     <NeutralLanguage>en-US</NeutralLanguage>
     <Copyright>Copyright © Octopus Deploy 2017</Copyright>
+    <LangVersion>8</LangVersion>
```

Comment:
Do we need this?

Reply:
Yes, because the project targets net40, so without this line it thinks C# 7.3 instead of 8 and nullable can't be enabled